media-libs/mesa/mesa-9999.ebuild

```diff
@@ -386,10 +394,14 @@ multilib_src_configure() {
 		gallium_enable video_cards_radeon r300 r600
 	fi
 
-	# opencl stuff
-	emesonargs+=(
-		-Dgallium-opencl="$(usex opencl icd disabled)"
-	)
+	if use opencl; then
+		PKG_CONFIG_LIBDIR="$(get_llvm_prefix)/$(get_libdir)/pkgconfig:${ESYSROOT}/usr/$(get_libdir)/pkgconfig"
```

Comment:
After quite a bit of debugging, I think this is wrong. `PKG_CONFIG_LIBDIR` is apparently the primary path, and `PKG_CONFIG_PATH` is the secondary path.

If you set `PKG_CONFIG_PATH` you don't have to include `${ESYSROOT}/usr/$(get_libdir)/pkgconfig` because you're not overwriting the primary path (which already contains that).